### PR TITLE
Toggle lightning spread in LightningMutation

### PR DESCRIPTION
**Pull request by PotatoStealer**

### CODE REVIEW

File: PGM/src/main/java/tc/oc/pgm/mutation/types/targetable/LightningMutation.java [hunk 1]
```diff
@@ -20,6 +21,16 @@ public LightningMutation(Match match) {
...
     }
 
+    @Override
+    public void enable() {
+        match().getWorld().setGameRuleValue("doFireTick", "false");
+    }
+
+    @Override
+    public void disable() {
+        match().getWorld().setGameRuleValue("doFireTick", "true");
```
Comment (Pablete1234) [inline PGM/src/main/java/tc/oc/pgm/mutation/types/targetable/LightningMutation.java]: Won't this mess with a map that has fireTick off by default?

**Comment by Electroid** [comment]
It is intentional for fire spread to be enabled.

**Comment by Pablete1234** [comment]
It can be defined via map XML if fire spread should be enabled or disabled. By enabling the mutation it makes sense that you may want to disable fire spread, but it should be reset to the map default when the mutation is disabled, not just set to true

**Comment by Electroid** [comment]
If a map wants to disable fire-spread, it can be added via the XML.

**Comment by Pablete1234** [comment]
It's already a switch on xml: https://docs.stratus.network/modules/gamerules
The issue is enabling and disabling the mutation will make the defined rule in the XML ignored, and fire spread will be always enabled.

**Pull request closed without merging**